python/rgb_hex.py

```diff
@@ -1,3 +1,5 @@
-rgb = lambda r, g, b: "{:02X}{:02X}{:02X}".format(
-    max(0, min(255, r)), max(0, min(255, g)), max(0, min(255, b))
-)
+def rgb(r, g, b):
+    r = max(0, min(255, r))
+    g = max(0, min(255, g))
+    b = max(0, min(255, b))
+    return "{:02X}{:02X}{:02X}".format(r, g, b)
```

Comment:
```suggestion
    return f"{r:02X}{g:02X}{b:02X}"
```

Would this work? Seem like it should, but not sure.